lib/src/operance_data_column_width.dart

```diff
@@ -1,39 +1,30 @@
-// 🐦 Flutter imports:
-import 'package:flutter/widgets.dart';
-
 /// A class representing the width of a column in the OperanceDataTable.
 class OperanceDataColumnWidth {
   /// Creates an instance of [OperanceDataColumnWidth].
   ///
-  /// The [size] and [factor] parameters are optional.
-  /// If [size] is provided, it will be used as the column width.
+  /// The [size] parameter is optional and takes precedence over [factor].
+  /// If [size] is provided, it will be used as the fixed column width.
   /// If [size] is not provided, the width will be calculated based on the
-  /// [factor].
+  /// [factor], which defaults to 0.15 (15% of the available width).
   const OperanceDataColumnWidth({
     this.size,
     this.factor = 0.15,
-  });
+  }) : assert(factor > 0 && factor <= 1, 'Factor must be between 0 and 1',);
```

Comment:
should be formatted, so it turns to 2 lines instead of the single line, that's the intention of the trailing comma, ensure to run the formatter before committing, i'll raise a ticket to run the formatter on git commit so it'll do it for us, but for now it'll be a manual process